addons/hr_holidays/wizard/hr_holidays_summary_department.py

```diff
@@ -2,31 +2,33 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 import time
 
-from openerp.osv import fields, osv
-from openerp.tools.translate import _
-from openerp.exceptions import UserError
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class HolidaysSummaryDept(models.TransientModel):
 
-class hr_holidays_summary_dept(osv.osv_memory):
     _name = 'hr.holidays.summary.dept'
     _description = 'HR Leaves Summary Report By Department'
-    _columns = {
-        'date_from': fields.date('From', required=True),
-        'depts': fields.many2many('hr.department', 'summary_dept_rel', 'sum_id', 'dept_id', 'Department(s)'),
-        'holiday_type': fields.selection([('Approved','Approved'),('Confirmed','Confirmed'),('both','Both Approved and Confirmed')], 'Leave Type', required=True)
-    }
 
-    _defaults = {
-         'date_from': lambda *a: time.strftime('%Y-%m-01'),
-         'holiday_type': 'Approved'
-    }
+    date_from = fields.Date(string='From', required=True, default=time.strftime('%Y-%m-01'))
```

Comment:
date_from will be the first day of the month when the server/worker was last restarted...